caffe2/operators/fully_connected_op_gpu.cc

```diff
@@ -67,41 +67,41 @@ bool RunFullyConnectedGradientOpOnCUDADevice(
         float, // dW
         float, // dB
         float>(); // Math
-  } else if (op->Input(0).template IsType<float16>()) {
-    if (float16_compute) {
+  } else if (op->Input(0).template IsType<at::Half>()) {
+    if (float_compute) {
```

Comment:
shouldn't this be called half_compute?

Reply:
I think I need to change them all back to float16_compute for bc

Reply:
that's fine.